### ISSUE TITLE
SessionController create route unreachable for LiveView-based update password flow

### ISSUE THREAD

**Issue by jsmestad** [phoenixframework/phoenix#4925]
### Background

I could be wrong about this, but I have walked the code several times and this does appear to be a logic error. I am doing this in an existing app, so I could be doing something wrong 🤞 

### Environment

* Elixir version (elixir -v): `1.14.0-rc.1`
* Phoenix version (mix deps): `1.18.0-dev`
* Operating system: macOS

### Expected behavior

Ability to invoke [this clause](https://github.com/phoenixframework/phoenix/blob/master/priv/templates/phx.gen.auth/session_controller.ex#L11-L15) `def create(conn, %{"_action" => "password_updated"} = params)` in `SessionController` when logged in so complete the Change Password flow provided by phx.gen.auth.

### Actual behavior

[This generated test](https://github.com/phoenixframework/phoenix/blob/master/priv/templates/phx.gen.auth/settings_live_test.exs#L93-L116) fails do to hitting the redirect guarding the route [here](https://github.com/phoenixframework/phoenix/blob/master/test/mix/tasks/phx.gen.auth_test.exs#L398).

If I move the `post "/users/log_in", UserSessionController, :create` outside of the `:require_if_user_is_authenticated` plug, the test passes as written.

**Comment by jsmestad** [comment]
Looking further into why `UserSessionControllerTest` has a passing test, it looks like [this test](https://github.com/phoenixframework/phoenix/blob/master/priv/templates/phx.gen.auth/session_controller_test.exs#L87-L100) is missing a call to `log_in_user(user)`.

**Comment by aptinio** [comment]
I tried to reproduce this, but all tests passed in a fresh phoenix project created with:

```sh
mix phx.new foo # with phoenix and phoenix_live_view pointing to github
mix phx.gen.auth --live Accounts User users
```

Are you still reproducing the issue, @jsmestad?

**Comment by Eiji7** [comment]
@chrismccord  After fix in #5067 I can confirm that all tests passes with and without `--no-gettext` option. :tada:

Do you think we can close this issue especially since it have over 2 months and nobody is able to reproduce this problem?
